ConwaysGameOfLife.Tests/SourceData.cs

```diff
@@ -0,0 +1,16 @@
+namespace ConwaysGameOfLife.Tests
+{
+    public static class SourceData
+    {
+        public static Universe UniverseInitializedAllDeadCells()
+        {
+            var initData = new CellState[][]
+            {
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead },
+                new []  {CellState.Dead,   CellState.Dead,    CellState.Dead }
+            };

```

Comment:
could we consider other options instead of a 2D array of cellstate, given that for this kata a List is used to store cells instead of a 2D array

you could have a helper function in here to create a list of cells which are passed to Universe in the constructor. we can talk about this option on monday//use it as a refactoring exercise

eg. input:
```csharp
var initData = "XXX" + Environment.NewLine +
"OOO" + Environment.NewLine +
XXX"
```

eg. output:
```csharp
var output = List<Cell> {
  new Cell(0, 0, CellState.Dead),
  new Cell(1, 0, CellState.Dead),
  new Cell(2, 0, CellState.Dead),
  new Cell(0, 1, CellState.Alive),
  new Cell(1, 1, CellState.Alive),
  new Cell(2, 1, CellState.Alive),
  new Cell(0, 2, CellState.Dead),
  new Cell(1, 2, CellState.Dead),
  new Cell(2, 2, CellState.Dead)
}
```